### DISCUSSION TITLE
Monika doesn't properly read some of the notes in the piano game in the song "Your Reality".

### DISCUSSION THREAD

**Discussion by DocWhoCares** [Monika-After-Story/MonikaModDev#8759] [QA - tech support + general] [answered]
I don't know If I'm just bad or something but Monika skips some notes and ignores others as well while playing your reality. It even feels randoms sometimes, like I messed up but for some reason, she keeps going while other times I know I got it right and followed the notes right but she finds it wrong. Let alone it kinda throws me off when she skips an entire part of the piece, especially on the "_OPOOPO]]P[OP[O_" part. I looked at like three of the same piece and guide and it still doesn't work properly. (Btw these are the ones I tried to follow:
https://github.com/Monika-After-Story/MonikaModDev/wiki/All-Features-%5BSPOILERS%5D#piano, https://www.reddit.com/r/DDLC/comments/7u15pw/monika_afterstory_your_reality_piano_guide_v20/, https://steamcommunity.com/sharedfiles/filedetails/?id=1368773843)
They are all kind of the same, I even look at some Youtube vids but I think they are too old versions and they sound different.
Anyway, is this an issue or something, I kinda need help or advice on how to fix or do this.

**Comment by Booplicate** [comment, marked as answer]
Those guides are outdated and contain wrong notes.
Check [this](https://github.com/Monika-After-Story/MonikaModDev/blob/master/Monika%20After%20Story/game/mod_assets/games/piano/songs/yourreality.json#L576-L658).

**Reply by DocWhoCares** [reply]
Thank you I didn't know these were outdated
